tests/api/v1/collectors/heartbeat.js

```diff
@@ -1017,7 +1018,43 @@ describe('tests/api/v1/collectors/heartbeat.js >', () => {
         .then(done)
         .catch(done);
       });
+    });
+
+    describe('added generator >', () => {
+      before(() => tu.toggleOverride('returnUser', true));
+      after(() => tu.toggleOverride('returnUser', false));
```

Comment:
"after" should restore it to whatever it was before, which is not necessarily guaranteed to be "false" -- safer to save the original value in the "before"

Reply:
We are not using the returnUser anywhere else in the test. Even if we are, the other tests ought to setup then teardown. Not sure if it is a good idea to add more code to this section.

Reply:
don't think it matters much either way.

Reply:
It's not about whether it's used in *this* test, but rather across all the tests.